packages/portalnetwork/src/networks/history/util.ts

```diff
@@ -294,13 +295,60 @@ export const verifyPreCapellaHeaderProof = (
     'blockHash',
   ])
   const reconstructedBlock = ssz.bellatrix.BeaconBlock.createFromProof({
```

Comment:
```suggestion
  const reconstructedBlock = ssz[fork[.BeaconBlock.createFromProof({
```

?

Reply:
In this instance, this function should only ever be called for post merge pre Capella blocks, so we can safely stick with the bellatrix types here.